modules/spam/text.py

```diff
@@ -344,10 +342,8 @@ def brainfuck(self, update: Update, context: CallbackContext) -> None:
                 if inputs:
                     data[data_ptr] = ord(inputs.pop(0))
                 else:
-                    update.message.reply_text(
-                        f"Error at position {instr_ptr}: `,` expected input but none was provided."
-                    )
-                    return
+                    # if no input is given, a 0 byte is read
+                    data[data_ptr] = 0

```

Comment:
You sure about that one?

Reply:
Yeee, maybe it should be an option? It's what https://copy.sh/brainfuck/ seems to do and it's handy cause it allows you to take input in a loop and stop when there's nothing more to read, with the current implementation, you need to know exactly how much input you expect or have a stop token or something

Reply:
I used this to write a little reverser script so you can do "/brainfuck [reverser] some arbitrary text" and it will return "txet yrartibra emos".

(This is a crucial feature that we cannot live without)

Reply:
Fair enough!